libs/api/database/src/lib/content/schemas/tags.schema.ts

```diff
@@ -11,10 +11,10 @@ export class TagsDocument extends Document implements TagsModel {
     @Prop({ trim: true, required: true, index: 'text' })
     name: string;
 
-    @Prop({ trim: true, required: true })
-    desc: string;
+    @Prop({ trim: true, default: undefined })
```

Comment:
no need for the `default: undefined`, I believe--pretty sure an optional (i.e., has a `?`) defaults to `null`. If we _do_ define a default value here, I think `null` would be better, as that would ensure it shows up in the JSON payload, and is a teeny bit more explicit.